docs/tutorials/gradients.ipynb

```diff
@@ -626,6 +626,12 @@
         "        pass\n",
         "\n",
         "    @tf.function\n",
+        "    def get_gradient_circuits(self, programs, symbol_names, symbol_values):\n",
+        "        \"\"\"See base class description.\"\"\"\n",
+        "        raise NotImplementedError(\n",
+        "            \"Gradient circuits are not implemented in this tutorial.\")\n",
```

Comment:
Should probably provide a meaninful description here other than "See base class".

Reply:
Expanded the docstring